content/en/docs/components/training/user-guides/mpi.md

```diff
@@ -28,7 +28,7 @@ cd mpi-operator
 kubectl apply -f deploy/v2beta1/mpi-operator.yaml
 ```
 
-Alternatively, follow the [getting started guide](https://www.kubeflow.org/docs/started/getting-started/) to deploy Kubeflow.
+Alternatively, follow the [getting started guide](https://www.kubeflow.org/docs/started/) to deploy Kubeflow.
```

Comment:
Its still not clear why this change is being made?

(its not in the pipelines section, and results in a 404)

Reply:
Reverted.